source/common/modules/markdown-editor/hooks/codeblock-classes.ts

```diff
@@ -29,8 +29,10 @@ const findCodeDebounced = debounce(findCode, 400, { leading: true })
  */
 export default function codeblockClassHook (cm: CodeMirror.Editor): void {
   cm.on('keyHandled', handleNewline)
```

Comment:
This hook shouldn't be needed anymore, right? If I remember correctly, it had something to do with the debouncing, and that you don't want to debounce the `Enter` key. But now that we go rid of the debouncing stuff, this hook seems to be redundant.